jax/numpy/lax_numpy.py

```diff
@@ -3320,6 +3320,12 @@ def unique(ar, return_index=False, return_inverse=False,
   raise NotImplementedError(
         "np.unique is not implemented for the axis argument")
 
+@_wraps(np.union1d)
+def union1d(ar1, ar2):
+  ar1 = ravel(asarray(ar1))
+  ar2 = ravel(asarray(ar2))
```

Comment:
Remove `asarray` in both cases here; `ravel()` already handles all valid inputs.

Reply:
Okay!